ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml

```diff
@@ -143,44 +134,46 @@ StatusScrollView {
 
                     StatusBaseText {
                         Layout.maximumWidth: mainLayout.width - Style.current.padding
-                        Layout.alignment: previewBox.leftAlignment ? Qt.AlignLeft : Qt.AlignHCenter
+                        Layout.alignment: Qt.AlignLeft
                         text: previewBox.value
                         elide: Text.ElideRight
                         font.pixelSize: Theme.primaryTextFontSize
                         color: Theme.palette.directColor1
-                        horizontalAlignment: previewBox.leftAlignment ? Text.AlignLeft : Text.AlignHCenter
+                        horizontalAlignment: Text.AlignLeft
```

Comment:
the `Text.AlignLeft` and `Qt.AlignLeft` are default; you might as well drop them